src/dosbox.cpp

```diff
@@ -544,6 +544,20 @@ void DOSBOX_Init() {
 	        "join      : combines them into one big section (legacy behavior).\n"
 	        "overwrite : use the last one encountered, like other conf settings.");
 
+	Pbool = secprop->Add_bool("automount", only_at_start, true);
+	Pbool->Set_help(
+	        "Mounts 'drives/[c]' directories as drives on startup, where [c] is\n"
+	        "a lower-case drive letter from a to y.  The 'drives' folder can be\n"
+	        "provided relative to the current directory or via built-in resources.\n"
+	        "Mount settings can be optionally provided using a [c].conf file along-\n"
+	        "-side the drive's directory, with content as follows:\n"
+	        "  [drive]\n"
+	        "  type    = dir, overlay, floppy, or cdrom\n"
+	        "  label   = custom_label\n"
+	        "  path    = path-specification, ie: path = %%path%%;c:\\tools\n"
```

Comment:
I was first unsure about having to manually specify `%%path%%`, which means by not specifying it one can undo earlier path configurations from different drive configs. But I guess this gives the user more flexibility on the path priorities.

Reply:
Yeah; I thought about taking a list of subdirectories, and inferring the drive letter itself, and then having another settings to prefix or postfix them into the PATH.. but in the end figured I would let the user define the PATH here just as they would in autoexec (more flexible and no new mechanisms).